sample-dcrlnfaucet.conf

```diff
@@ -23,6 +23,11 @@ bind_addr=:80
 ; use the lets encrypt service to get a certificate for it.
 use_le_https=false
 
+; Use the XRealIP to get the real client IP, but only if a reverse
+; proxy or load balancer is correctly setting the X-Forwarded-For and/or
+; X-Real-Ip headers. (Default is false.)
+userealip=0
+
```

Comment:
We should add the other new config item (`ActionsTimeLimit`) to the sample file as well.